src/components/Playground/Playground.tsx

```diff
@@ -13,9 +14,11 @@ import Image from 'next/image';
 import { usePlayground } from './hooks/usePlayground';
 
 export default function Playground() {
-  const { decreaseCommandsLeft, search, timeLeft, commandsLeft, setSearch } =
+  const { decreaseCommandsLeft, timeLeft, commandsLeft } =
     usePlayground();
 
+  const MemoizedSearchBox = useMemo(() => <SearchBox />, []);
```

Comment:
This seems like a hack we had to do because we don't have separate components for the 2 sections of playground (The Terminal and the Searchbox). 

Can we just create a component for the Left Column of Playground (Terminal + Commands Left etc), and call usePlayground from it? Basically, neither the Searchbox or any of it's parents should care about the `time` or `commandsLeft`, these should be state of the sibling for Searchbox.